### PR TITLE
Adds plunger unicorns

### PR DESCRIPTION
**Pull request by Time-Green**
You can now wear plungers on your face to be a plunger unicorn.

:cl: Chef spooky bones
sprite: HIGHLY REQUESTED: You can now wear plungers on your face!
/:cl:

### CODE REVIEW

File: code/game/objects/structures/lavaland/geyser.dm [hunk 1]
```diff
@@ -56,6 +56,8 @@
 	icon = 'icons/obj/watercloset.dmi'
 	icon_state = "plunger"
 
+	slot_flags = ITEM_SLOT_MASK
```
Comment (Jalleo) [inline code/game/objects/structures/lavaland/geyser.dm]: Please give it welder vision. You are putting it over your face

**Review by Jalleo — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/lavaland/geyser.dm (hunk 1 above)
Comment (Time-Green) [reply code/game/objects/structures/lavaland/geyser.dm]: light is absorbed through the handle and deposited into your eyes through the advanced photon conducting red suction cup

**Comment by GuyonBroadway** [comment]
Make it so tossing one at someone has a chance to get stuck on thier face.

**Comment by FloranOtten** [comment]
If shot out of an air canister gun, it has a 100% chance of sticking.

**Comment by kriskog** [comment]
Make it only work on bald people :smiling_imp:

**Comment by fludd12** [comment]
Make it 🚲🏠

**Comment by OverlyGeneric** [comment]
allow it to be put onto ian.

do it.

**Comment by wesoda25** [comment]
tg is saved

**Comment by kriskog** [comment]
> 
> 
> Make it only work on bald people smiling_imp

https://i.imgur.com/HkKDYSW.mp4

**Comment by Tlaltecuhtli** [comment]
give it embedding

**Comment by 81Denton** [comment]
Finally some good content

**Pull request merged**